lib/test/cdo/test_share_filtering.rb

```diff
@@ -11,39 +11,77 @@ def setup
     Geocoder::Lookup::Test.set_default_stub([{types: []}])
   end
 
+  # @param title_name [String] The name of the title of the program.
+  # @param title_text [String] The text of the title of the program.
+  # @return [String] A sample program.
+  def get_program(title_name, title_text)
+    '<xml><block type="when_run" deletable="false">'\
+      '<next><block type="studio_showTitleScreen">'\
+      "<title name=\"TITLE\">#{title_name}</title>"\
+      "<title name=\"TEXT\">#{title_text}</title>"\
+      '</block></next></block></xml>'
+  end
```

Comment:
nit: `get_program` implies that it's retrieving something rather than constructing it. How about `mock_program`, or `generate_program`?